src/transforms/dodge.d.ts

```diff
@@ -1,20 +1,56 @@
 import type {ChannelValueSpec} from "../channel.js";
 import type {Initialized} from "./basic.js";
 
-export type DodgeAnchor = "top" | "right" | "bottom" | "left" | "middle";
+export type DodgeAnchorX = "right" | "left" | "middle";
 
-export interface DodgeOptions {
-  x?: ChannelValueSpec;
+export type DodgeAnchorY = "top" | "bottom" | "middle";
+
+export interface DodgeOptionsX {
+  /** The vertical position of the mark. */
   y?: ChannelValueSpec;
+  /** The radius of the mark. */
```

Comment:
This is tricky because when a channel, as opposed to a constant radius number, the channel values typically represent areas instead of radii due to the default *r* *sqrt* scale.

Reply:
the size?